<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableViewDialog</class>
 <widget class="QDialog" name="TableViewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../controller.qrc">
    <normaloff>:/images/controller.ico</normaloff>:/images/controller.ico</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>9</y>
     <width>621</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="userEventTab">
       <attribute name="title">
        <string>Swiping card records</string>
       </attribute>
       <widget class="QTableView" name="userEventTableView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>621</width>
          <height>391</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </widget>
      <widget class="QWidget" name="sessionTab">
       <attribute name="title">
        <string>Using records</string>
       </attribute>
       <widget class="QTableView" name="sessionTableView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>621</width>
          <height>391</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </widget>
      <widget class="QWidget" name="systemEventTab">
       <attribute name="title">
        <string>System records </string>
       </attribute>
       <widget class="QTableView" name="systemEventTableView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>621</width>
          <height>391</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="prevButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
        <property name="icon">
         <iconset resource="../controller.qrc">
          <normaloff>:/images/arrow_left.png</normaloff>:/images/arrow_left.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pageLabel">
        <property name="text">
         <string>0/0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
        <property name="icon">
         <iconset resource="../controller.qrc">
          <normaloff>:/images/arrow_right.png</normaloff>:/images/arrow_right.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
        <property name="icon">
         <iconset resource="../controller.qrc">
          <normaloff>:/images/btn_close.png</normaloff>:/images/btn_close.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../controller.qrc"/>
 </resources>
 <connections/>
</ui>
